webapp/sources/pom.xml

```diff
@@ -181,6 +186,8 @@ limitations under the License.
             <arg>-Ycache-plugin-class-loader:last-modified</arg> <!-- Enables caching of classloaders for compiler plugins -->
             <arg>-Ycache-macro-class-loader:last-modified</arg>  <!-- and macro definitions. This can lead to performance improvements. -->
             <arg>-P:silencer:checkUnused</arg>
+<!--            <arg>-P:wartremover:only-warn-traverser:org.wartremover.warts.StringPlusAny</arg>-->
+<!--            <arg>-P:wartremover:only-warn-traverser:org.wartremover.warts.ToString</arg>-->
```

Comment:
do you want to keep these commented to use it when needed easily ?

Reply:
yes, I want to be able to test from time to time, at least until that PR is merged. They lead to a lot of false positive, so we can't keep them always on, but they are the only way to find an implicit `toString` on ID which is no more what we need